sdk/appcenter-analytics/src/test/java/com/microsoft/appcenter/analytics/AnalyticsTest.java

```diff
@@ -863,6 +865,24 @@ public void createTransmissionTargetBeforeStart() {
         AppCenterLog.error(anyString(), contains("AppCenter is not configured"));
     }
 
+    @Test
+    public void triggerInterval() {
+        Analytics analytics = Analytics.getInstance();
+        boolean result = Analytics.setTransmissionInterval(1);
+        assertFalse(result);
```

Comment:
I think we should split this test into separate.
In this test we are checking:
**On disabled module**
1) Setting less than min value (false)
2) Setting more than max value (false)
3) Setting value (true)
**On enabled module**
4) Setting less than min value (false)
It would be great if each test had one and only reason to fail, separate for each test. And, the latter (4th in a list), as pointed out by @guperrot, is not quite a valid test.